### PR TITLE
Loading seed into test db

### PR DESCRIPTION
**Pull request by lwrubel**
## Why was this change made? 🤔

To load the required User seed into the test database. 

## How was this change tested? 🤨
Running tests

⚡ ⚠ If this change involves consuming from or writing to another service (or shared file system), ***run [integration test create_object_h2_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test manually in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -41,6 +41,7 @@
   exit 1
 end
 
+Rails.application.load_seed # loading seeds
```
Comment (jcoyne) [inline spec/rails_helper.rb:44]: Doesn't this cause an error the second time you run the suite?

Comment (lwrubel) [reply spec/rails_helper.rb]: Yes, I'm seeing it does in Circle CI but not locally.

Comment (lwrubel) [reply spec/rails_helper.rb]: Should we instead put into the README to run ` bundle exec RAILS_ENV=test rake db:seed`?

Comment (jcoyne) [reply spec/rails_helper.rb]: sure, but `RAILS_ENV=test bin/rails db:seed` is how I'd do it.

**Pull request closed without merging**